src/component/app/index.tsx

```diff
@@ -104,7 +96,7 @@ function AuthenticatedRoute() {
   return (
     <UserContext.Provider value={userContext}>
       <Routes>
-        <Route path="profile" element={<Profile userLoading={userLoading} />} />
+        <Route path="profile" element={<Profile userLoading={false} />} />
```

Comment:
Must be `false` due to the above checks. Is this intended?

Reply:
nope, not intended.

Reply:
We have
```js
  } else if (userLoading || userStatus === "idle") {
    return <span>User loading</span>;
  }
```
above. Should we drop that to let the `Profile` component render whether the user is loading or not?